### ISSUE TITLE
Question about models.py and the result

### ISSUE THREAD

**Issue by supermark1995** [koshian2/OctConv-TFKeras#13]
I have a question about the code and the result
First,the code in models.py line 42 to 46 and line 48 to 52.These two range of code is same.Is that coding wrong or it is necessary to do it ?
Second, I ran your code and my val_loss is always larger than 0.3, is that correct?

**Comment by koshian2** [comment]
@supermark1995 
(1) Conv layer is required twice for each module due to ResNet structure.
(2) I don't remember detailed loss values. Isn't val loss lowered by executing according to this code without additional data augmentation ,regularization or learning rate changes?

**Comment by supermark1995** [comment]
@koshian2 
Excuse me, I had a "Floating point exception" when I train octave convolution with alpha = 1,and I saw that you have a annotation in oct_conv2d.py line 19 -- "alpha: Low channel ratio (alpha=0 -> High only, alpha=1 -> Low only)". So is that OK when I set alpha equal one when I training the code.If yes,do you have any idea to fix this bug?
